cafebabel/templates/base.html

```diff
@@ -79,13 +76,12 @@
 </main>
 
 <footer>
-  <p>© Cafébabel — {{ get_year() }}</p>
+  <p>© Cafébabel — 2017</p>
```

Comment:
I think you failed a merge here.

Reply:
ah ok, I fixed it. But there were other chances regarding this chance, right ?

Reply:
changes*, change*